### DISCUSSION TITLE
闻香识女人 | /posts/scent-of-a-woman/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#577] [Announcements]
## 闻香识女人

https://yuanfan.rbind.io/posts/scent-of-a-woman/

**Comment by residualsun1** [comment]
说到《闻香识女人》，让我想起高中有一段时间和朋友很痴迷一些外国经典电影，比如《海上钢琴师》、《天堂电影院》、《西西里的美丽传说》、《布达佩斯大饭店》、《美国往事》和《死亡诗社》（《闻香识女人》我们也看了）。我们那时是集中时间（花了几个星期）看完了上面所有电影，现在想想感觉像是趁着没忙着应付生活，幸运地有过一段美丽体验。前段时间我还在读书小组里听到有位朋友说，“电影是生命的另一种延续”，看到你聊起《闻香识女人》，过去的记忆都慢慢浮现了起来，但上大学后竟然没怎么看过电影了，我打算再回温一遍这些电影。

**Reply by earfanfan** [reply]
>电影是生命的另一种延续

关于这点，我也有看到过较为相似的说法，“写作是品尝生活第二次”。

你提到的这些电影对我来说都是鼎鼎大名、如雷贯耳，但都是还没看过的，以后有时间我也会慢慢看的。哈哈，终于出现了可以跟我讨论下电影的读者了，俺要偷着乐一会。

对了，我还没看到你把你的博客地址发到 COS 论坛呢，打算撒时候去[那里](https://d.cosx.org/d/421538/160)留下你的足迹呢

**Reply by residualsun1** [reply]
> “写作是品尝生活第二次”

喜欢这个表达！

> 你提到的这些电影对我来说都是鼎鼎大名、如雷贯耳，但都是还没看过的，以后有时间我也会慢慢看的。哈哈，终于出现了可以跟我讨论下电影的读者了，俺要偷着乐一会。

能和别人分享这些电影本身也是很愉悦的过程！希望在未来的某一天，它们中的某一部或许会给你的生活增添几分恬静~

> 打算撒时候去那里留下你的足迹呢

我还是有点点不好意思。在统计之都逛了有一段时间，觉得大家的博客都很独特，也很迷人（虽然外观是直接的，但还得是故事本身）！我之前在Hugo的模板网站逛了好久，但都找不到符合自己心意的，直到看到了你的博客。感谢你和主题开发者的开源代码~但我想还是先继续耕耘内容，等我的博客知识积累得更多，我可以继续修改主题，让它更有自己的特色。不过说不准写得多了，我一样能感受到明显的属于自己精神家园的气息，相信那时候我也会有底气和大家分享自己的足迹。
